sys-libs/llvm-libunwind/files/Makefile

```diff
@@ -0,0 +1,43 @@
+ASFLAGS+=-Wa,--noexecstack
```

Comment:
What's the reason you're reinventing the build system instead of using upstream's?